### PR TITLE
feat: add Console local network guide

### LINKED ISSUE #69: Create Ash Console local network guide

### PR DESCRIPTION
**Pull request by leopaul36**
### Linked issues

- Fix #69 

### Changes

- Add Avalanche Local network guide

### Additional comments

- Prerequisites, Cloud credential setup and project setup are duplicated from the Avalanche Fuji node guide

### CODE REVIEW

File: docs/console/guides/fuji-node/avalanche-node.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The Ash Console is currently in alpha and **not production-ready**. It is under
 
 We can finally deploy our Avalanche node [resource](/docs/console/glossary#resource)!
 
-1. Gather the node ID secret ID and cloud credentials secret ID from the previous steps.
+1. Gather the node ID secret ID and cloud region secret ID from the previous steps.
```
Comment (Nuttymoon) [inline docs/console/guides/fuji-node/avalanche-node.md:13]:
```suggestion
1. Gather the node ID secret ID and cloud region ID from the previous steps.
```

File: docs/console/guides/local-network/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+sidebar_position: 2
+---
+
+# Avalanche local network
```
Comment (Nuttymoon) [inline docs/console/guides/local-network/index.md:5]:
```suggestion
# Avalanche Devnet (Local Network)
```

File: docs/console/guides/local-network/index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+:::caution
+The Ash Console is currently in alpha and **not production-ready**. It is under active development and subject to breaking changes.
+:::
+
+This guide will walk you through setting up an 5-node Avalanche [local](https://docs.avax.network/nodes/configure/avalanchego-config-flags#network-id) network with the Ash Console.
```
Comment (Nuttymoon) [inline docs/console/guides/local-network/index.md:11]:
```suggestion
This guide will walk you through setting up a 5-node Avalanche devnet ([local network](https://docs.avax.network/nodes/configure/avalanchego-config-flags#network-id) network) with the Ash Console.

Devnets allow you to develop your Subnet out of sight in an environment where you have full control!

:::info
Devnets use the `local` network setup with hardcoded validator nodes and airdrop address.
:::
```

File: docs/console/guides/local-network/node-ids.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+# 2. Node ID Secrets Generation
+
+:::caution
+The Ash Console is currently in alpha and **not production-ready**. It is under active development and subject to breaking changes.
+:::
+
+The Ash Console **decorelates the node ID from Avalanche nodes**, so that you can have a better control over your validator nodes' lifecycle.
+
+Node IDs are stored as a [secret](/docs/console/glossary#secret) in the Console.
```
Comment (Nuttymoon) [inline docs/console/guides/local-network/node-ids.md:13]:
```suggestion
Node IDs are stored as [secrets](/docs/console/glossary#secret) in the Console.
```

File: docs/console/guides/local-network/node-ids.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+## Fetch the 5 node IDs certificates
+
+The ["Local Test Network Creation"](/docs/toolkit/ansible-avalanche-collection/tutorials/local-test-network) tutorial uses the same certificates for the 5 nodes of the local network. In this step, we will fetch the certificates files from the [`ansible-avalanche-getting-started`](https://github.com/AshAvalanche/ansible-avalanche-getting-started) repository.
+
+1. If not already done, create a folder for this guide and navigate to it:
+
+   ```bash
+   mkdir -p ash-console-guides/local-network
+   cd ash-console-guides/local-network
```
Comment (Nuttymoon) [inline docs/console/guides/local-network/node-ids.md:29]:
```suggestion
   mkdir -p ash-console-guides/devnet-network
   cd ash-console-guides/devnet-network
```

File: docs/console/guides/local-network/node-ids.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+:::caution
+The Ash Console is currently in alpha and **not production-ready**. It is under active development and subject to breaking changes.
+:::
+
+The Ash Console **decorelates the node ID from Avalanche nodes**, so that you can have a better control over your validator nodes' lifecycle.
+
+Node IDs are stored as a [secret](/docs/console/glossary#secret) in the Console.
+
+In this part of the guide, we will create **5 node ID secrets** for the 5 nodes of our local Avalanche network.
```
Comment (Nuttymoon) [inline docs/console/guides/local-network/node-ids.md:15]:
```suggestion
In this part of the guide, we will create **5 node ID secrets** for the 5 nodes of our Avalanche devnet.
```

File: docs/console/guides/local-network/node-ids.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+:::tip
+The Node ID secrets need to match the hardcoded Node IDs in the [`genesis_local.json`](https://github.com/ava-labs/avalanchego/blob/master/genesis/genesis_local.json#L47) file of the [AvalancheGo](https://github.com/ava-labs/avalanchego) codebase.
+:::
+
+## Fetch the 5 node IDs certificates
+
+The ["Local Test Network Creation"](/docs/toolkit/ansible-avalanche-collection/tutorials/local-test-network) tutorial uses the same certificates for the 5 nodes of the local network. In this step, we will fetch the certificates files from the [`ansible-avalanche-getting-started`](https://github.com/AshAvalanche/ansible-avalanche-getting-started) repository.
```
Comment (Nuttymoon) [inline docs/console/guides/local-network/node-ids.md:23]:
```suggestion
The [Local Test Network Creation tutorial](/docs/toolkit/ansible-avalanche-collection/tutorials/local-test-network) uses the same certificates and keys for the 5 validator nodes. Let's fetch the certificate and key files from the [`ansible-avalanche-getting-started`](https://github.com/AshAvalanche/ansible-avalanche-getting-started) repository.
```

File: docs/console/guides/local-network/node-ids.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+1. If not already done, create a folder for this guide and navigate to it:
+
+   ```bash
+   mkdir -p ash-console-guides/local-network
+   cd ash-console-guides/local-network
+   ```
+
+2. Fetch the local nodes certificates from the [`ansible-avalanche-getting-started`](https://github.com/AshAvalanche/ansible-avalanche-getting-started/tree/main/files/staking) repository:
```
Comment (Nuttymoon) [inline docs/console/guides/local-network/node-ids.md:32]:
```suggestion
2. Fetch the local nodes certificates and keys from the [`ansible-avalanche-getting-started`](https://github.com/AshAvalanche/ansible-avalanche-getting-started/tree/main/files/staking) repository:
```

File: docs/console/guides/local-network/project-region.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+```bash title="Output"
+Project created successfully!
++--------------------------------------+-------------+---------------------+---------+---------------+------------------+
+| Project ID                           | Owner ID    | Name                | Network | Cloud regions | Created at       |
++======================================+=============+=====================+=========+===============+==================+
+| 70228a47-5ab1-493b-b921-5c22b83bfbf7 | fce8...3695 | local-network-guide | Local   |               | 2023-11-23T15:08 |
++--------------------------------------+-------------+---------------------+---------+---------------+------------------+
+
+Switched to project 'local-network-guide' (70228a47-5ab1-493b-b921-5c22b83bfbf7)!
+```
```
Comment (Nuttymoon) [inline docs/console/guides/local-network/project-region.md:30]: Suggestion: ~`local-network`~ `devnet`
```bash title="Command"
ash console project create '{"name": "devnet-guide", "network": "local"}'
```

```bash title="Output"
Project created successfully!
+--------------------------------------+-------------+--------------+---------+---------------+------------------+
| Project ID                           | Owner ID    | Name         | Network | Cloud regions | Created at       |
+======================================+=============+=====================+=========+===============+==================+
| 70228a47-5ab1-493b-b921-5c22b83bfbf7 | fce8...3695 | devnet-guide | Local   |               | 2023-11-23T15:08 |
+--------------------------------------+-------------+--------------+---------+---------------+------------------+

Switched to project 'devnet-guide' (70228a47-5ab1-493b-b921-5c22b83bfbf7)!
```

File: docs/toolkit/ansible-avalanche-collection/tutorials/vm-management-custom.md
Comment (Nuttymoon) [inline docs/toolkit/ansible-avalanche-collection/tutorials/vm-management-custom.md:1]: I think you commited this file by error.

**Review by Nuttymoon — CHANGES_REQUESTED** [review]

File: docs/toolkit/ansible-avalanche-collection/tutorials/vm-management-custom.md
Comment (leopaul36) [reply docs/toolkit/ansible-avalanche-collection/tutorials/vm-management-custom.md:1]: Removed

**Pull request merged**